site/docs/plugins/chat-members.md

```diff
@@ -0,0 +1,132 @@
+# Chat Members Plugin (`chat-members`)
+
+Automatically store information about users on a chat and retrieve it easilly
+
+## Introduction
+
+In many situations, it is necessary for a bot to have information about all the users of a given chat.
+Currently, though, the Telegram Bot API exposes no method that allows us to retrieve this information.
+
+This plugin comes to the rescue: automatically listening to `chat_member` events and storing the `ChatMember` object.
+It then allows you to use `ctx.chatMembers.getChatMember()` to obtain information about a specific chat member, taking the previously stored information into account.
```

Comment:
This is odd. First we say that there are no methods to keep track of the list. Then we give an example for retrieving info about a single member. As a reader, I'd ask myself why we can't just use `getChatMember` to do this, and what happened to the list of members.

Reply:
I'll remove this line.